packages/ipfs-core/src/components/pin/remote/ls.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const withTimeoutOption = require('ipfs-core-utils/src/with-timeout-option')
+
+module.exports = ({ serviceRegistry }) => {
+  /**
+   * List objects that are pinned by a remote service.
+   *
+   * @param {Query & AbortOptions} options
+   * @returns {AsyncGenerator<Pin>}
+   */
+  async function * ls (options) {
+    const { service } = options
+    const svc = serviceRegistry.serviceNamed(service)
```

Comment:
Unless I'm missing something this will return `null` unload config was loaded which may not be the case here.